Userland/Utilities/less.cpp

```diff
@@ -216,8 +217,18 @@ class Pager {
         }
     }
 
+    size_t calculate_digits_count(size_t line_numbers)
```

Comment:
`calculate_digits_count` is kind of an awkward name - `calculate_digits_in_number` or `count_digits_in_number` or something sounds better to me.